navigation/ca_hector/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ca_hector)
+
+find_package(catkin REQUIRED COMPONENTS
+  hector_geotiff
+  hector_mapping
+)
+
+catkin_package()
```

Comment:
I just realized that this package is not installing anything. How is the linter not complaining about it?